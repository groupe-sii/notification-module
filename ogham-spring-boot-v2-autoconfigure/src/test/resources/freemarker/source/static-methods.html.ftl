<!DOCTYPE html>
<html>
    <head>
        <title>Static method calls</title>
        <meta charset="utf-8" />
    </head>
    <body>
        <h1 class="title">${statics['mock.UtilityClass'].hello(name)}</h1>
        <p>${statics['mock.UtilityClass$NestedClass'].hello(name)}</p>
    </body>
</html>
