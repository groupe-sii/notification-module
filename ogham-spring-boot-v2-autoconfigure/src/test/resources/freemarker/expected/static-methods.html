<!DOCTYPE html>

<html>
    <head>
        <title>Static method calls</title>
        <meta charset="utf-8" />
    </head>
    <body>
        <h1 class="title">hello world</h1>
        <p>Hello world</p>
    </body>
</html>
